riptide-opentracing/README.md

```diff
@@ -63,29 +63,41 @@ By default a new span will be started for each request and it will be activated.
+| Tag/Log Field                | Decorator                          | Example                           |
+|------------------------------|------------------------------------|-----------------------------------|
+|                              | `CallSiteSpanDecorator`            | `admin=true`                      |
+| `component`                  | `ComponentSpanDecorator`           | `Riptide`                         |
+| `message` (log)              | `ErrorMessageSpanDecorator`*       | `Connection timed out`            |
+| `error`                      | `ErrorSpanDecorator`               | `false`                           |
+| `error.kind` (log)           | `ErrorSpanDecorator`               | `SocketTimeoutException`          |
+| `error.object` (log)         | `ErrorSpanDecorator`               | (exception instance)              |
+| `stack` (log)                | `ErrorStackSpanDecorator`          | `SocketTimeoutException at [...]` |
+| `http.content_language`      | `HttpContentLanguageSpanDecorator` | `en-DE`                           |
+| `http.content_length` (log)  | `HttpContentLengthSpanDecorator`   | `1337`                            |
+| `http.method_override`       | `HttpMethodOverrideSpanDecorator`  | `GET`                             |
+| `http.method`                | `HttpMethodSpanDecorator`          | `POST`                            |
+| `http.url`                   | `HttpUrlSpanDecorator`*            | `https://www.github.com/users/me` |
+| `http.path`                  | `HttpPathSpanDecorator`            | `/users/{user_id}`                |
+| `http.prefer`                | `HttpPreferSpanDecorator`          | `respond-async`                   |
+| `http.retry_after` (log)     | `HttpRetryAfterSpanDecorator`      | `120`                             |
+| `http.status_code`           | `HttpStatusCodeSpanDecorator`      | `200`                             |
+| `http.warning`               | `HttpWarningSpanDecorator`         | `110 - "Response is Stale"`       |
+| `peer.address`               | `PeerSpanDecorator`                | `www.github.com:80`               |
+| `peer.hostname`              | `PeerSpanDecorator`                | `www.github.com`                  |
+| `peer.port`                  | `PeerSpanDecorator`                | `80`                              |
+| `rate_limit.limit` (log)     | `RateLimitSpanDecorator`           | `100`                             |
+| `rate_limit.remaining` (log) | `RateLimitSpanDecorator`           | `99`                              |
+| `rate_limit.reset` (log)     | `RateLimitSpanDecorator`           | `3600`                            |
+| `retry`                      | `RetrySpanDecorator`*              | `true`                            |
+| `retry_number` (log)         | `RetrySpanDecorator`*              | `3`                               |
+|                              | `ServiceLoaderSpanDecorator`       |                                   |
```

Comment:
RetrySpanDecorator marked with * so reader can expect some explanation after the table, but this one doesn't marked with anything and got special behaviour and representation in this table. A bit confusing, unless you'll read the whole table and find out the special behaviour description

Reply:
Any idea how to improve that? Footnote style with superscript numbers?

Reply:
Footnote with numbers or star markers will make this much better